external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -92,11 +94,19 @@ private void ack(Tuple tuple) throws SolrServerException, IOException {
         if (commitStgy == null) {
             collector.ack(tuple);
         } else {
-            toCommitTuples.add(tuple);
-            commitStgy.update();
-            if (commitStgy.commit()) {
+            if (TupleUtils.isTick(tuple)) {
+                LOG.debug("TICK! forcing solr client commit");
+                collector.ack(tuple);
```

Comment:
Tick tuples are not generated from a spout but inserted periodically into the executor receive-queue. Since tick-tuples are not tracked by the acker, I dont think it needs to be acked. Acking it does not cause any harm but unnecessary. Correct me if I am overlooking something.